e2e-tests/k8s-e2e-bootstraping.sh

```diff
@@ -77,11 +77,11 @@ minikube version
 [ -n "$E2E_SETUP_KUBECTL" ] && setup_kubectl
 
 export MINIKUBE_WANTREPORTERRORPROMPT=false
-export MINIKUBE_HOME=$HOME
-export CHANGE_MINIKUBE_NONE_USER=true
-mkdir "$HOME"/.kube || true
-touch "$HOME"/.kube/config
-export KUBECONFIG=$HOME/.kube/config
+#export MINIKUBE_HOME=$HOME
```

Comment:
if they are not needed can we remove them instead of commenting? Or they are useful to test locally?

Reply:
I can try to move them to some of the `setup_` steps that are not used in the CI. That way we should be able to keep using them locally